### DISCUSSION TITLE
Sobre o Front in POA 2016

### DISCUSSION THREAD

**Discussion by eberhara** [frontendbr/forum#328] [Discussão]
Pessoal,

Alguém aqui vai na edição deste ano do [front in POA](http://frontinpoa.com.br/2016/), ou já foi em alguma edição anterior?

Gostaria de ter algum feedback do evento, pois parece ser muito bom!

**Comment by diessica** [comment]
Eu geralmente vou. É um ótimo evento, sim! Vale a pena!

**Comment by robsonbittencourt** [comment]
Eu fui na edição de 2014 se não me falha a memória. O evento foi bem bom. Ótimas palestras. Só faltou o café no coffee break :)

**Comment by expalmer** [comment]
Opa. Eu fui nos últimos 3 ou 4 anos e sempre foram bem relevantes.

**Comment by milenevlacerda** [comment]
Fui no ano de 2014 e só não fui no de 2015 por imprevistos... É um ótimo evento,  gosto muito :)

**Comment by karlprieb** [comment]
Fui em alguns e são bons, mas depois que o Felipe e o Jaydson começaram a monopolizar os eventos aqui em Porto Alegre eu parei de ir. Os valores aumentaram muito e todos os eventos de peso agora são organizados por eles :(

**Comment by eberhara** [comment]
@karlprieb Olhando as palestras deste ano, fiquei com esta mesma impressão.
